ROADMAP.md

```diff
@@ -14,30 +14,26 @@ The following features are considered for the near future:
 
 * **Windows support improvements**
 Antrea [supports Windows K8s Node](docs/windows.md) since version 0.7.0.
-However, a few features like network flow export are not supported for Windows
-Node yet. We will continue to add more features for Windows, and improve Antrea
-Agent and OVS installation on Windows Nodes.
+However, a few features including: Egress, NodePortLocal, IPsec encryption are
+not supported for Windows Node yet. We will continue to add more features for
+Windows, and improve Antrea Agent and OVS installation on Windows Nodes.
 
-* **IPv6 Pod network**
-Support IPv6 and IPv4/IPv6 dual-stack for Pod network. Right now Antrea supports
-only IPv4.
-
-* **Antrea NetworkPolicy**
-Antrea has started adding support for [Antrea native NetworkPolicy](docs/antrea-network-policy.md)
+* **Antrea NetworkPolicy enhancements**
+Antrea added support for [Antrea native NetworkPolicy](docs/antrea-network-policy.md)
 in addition to K8s NetworkPolicy since version 0.8.0. We already support
-ClusterNetworkPolicy and tiering, and will continue to add more NetworkPolicy
-extensions, like traffic logging, policy statistics, policy realization status,
-more matching criteria and actions, and external endpoints.
+Antrea (Namespace scoped) NetworkPolicy, ClusterNetworkPolicy, ClusterGroup, and
+tiering, but will continue to add more advanced NetworkPolicy features.
 
 * **Network diagnostics and observability**
 Network diagnostics and observability is one area we want to focus on. Antrea
 already implements some useful features on this front, including [Octant UI
 plugin](docs/octant-plugin-installation.md), [CLI](docs/antctl.md),
 [Traceflow](docs/traceflow-guide.md), [network flow export and visualization](docs/network-flow-visibility.md),
 [Prometheus metrics](docs/prometheus-integration.md), [OVS flow dumping](docs/antctl.md#dumping-ovs-flows)
-and [packet tracing](docs/antctl.md#ovs-packet-tracing). We will continue to
-enhance existing features and add new features to help diagnose K8s networking
-and NetworkPolicy implementation, and to provide good visibility into the Antrea
+and [packet tracing](docs/antctl.md#ovs-packet-tracing), [NetworkPolicy
+diagnosis] (docs/antctl.md#networkpolicy-commands). We will continue to enhance
```

Comment:
I would suggest "NetworkPolicy diagnostics" and not "NetworkPolicy diagnosis"

Reply:
Sure.